apiserver/facades/client/spaces/spaces.go

```diff
@@ -291,6 +317,73 @@ func (api *API) ListSpaces() (results params.ListSpacesResults, err error) {
 	return results, nil
 }
 
+func (api *APIv5) ShowSpace(_, _ struct{}) {}
+
+// ListSpaces lists all the available spaces and their associated subnets.
+func (api *API) ShowSpace(entities params.Entities) (params.ShowSpaceResults, error) {
+	canRead, err := api.authorizer.HasPermission(permission.ReadAccess, api.backing.ModelTag())
+	if err != nil && !errors.IsNotFound(err) {
```

Comment:
Can we add a note, saying that when the error is not found, we expect `canRead` to return false, which will then result in a `ErrPerm`. Although, I'm never sure that's a good way to model the side effect that `canRead` will be false by default.

Reply:
As this is used in multiple places in spaces, as well as in different places. I would suggest adding a comment to the method definition itself.
What do you think?

Else this would just be a comment on this method and all other places need to be updated as well.